spec/views/accounts/_identity_verification.html.erb_spec.rb

```diff
@@ -41,6 +42,17 @@
         strip_tags(t('account.index.verification.finish_verifying_html', sp_name: gpo_sp_name)),
       )
     end
+
+    context 'with vtr values' do
+      let(:vtr) { ['C2'] }
+      let(:acr_values) { nil }
+
+      it 'references initiating sp with prompt to finish verifying their identity' do
```

Comment:
Does 'references initiating SP in prompt instructing user to complete identity verification' read better?

Reply:
i don't think it reads better, and since i didn't author this initially, i am disinclined to update it.